src/features/sarcophagi/components/CleanButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button, Tooltip } from '@chakra-ui/react';
+import { useCleanSarcophagus } from 'hooks/thirdPartyFacet/useCleanSarcophagus';
+import { Sarcophagus } from 'types';
+
+export function CleanButton({ sarco }: { sarco: Sarcophagus }) {
+  const { clean, isCleaning, isError, mayFail } = useCleanSarcophagus(sarco.id);
+
+  return (
+    <>
+      {!isError ? (
+        <Tooltip
+          placement="top"
+          label="Deactivate the Sarcophagus and claim back culprit archaeologist locked bonds and digging fees"
```

Comment:
thinking for language here --  "Claim bonds and digging fees from Archaeologists that did not participate in the unwrapping ceremony."

Worrired the word "Deactivate" may confuse users